### DISCUSSION TITLE
Does `supabase auth` fully support multitenancy?

### DISCUSSION THREAD

**Discussion by nandox5** [supabase/supabase#10625] [Questions]
I'm debating if i should switch from next-auth to supabase auth.
Do you guys know if supabase auth fully support multi tenancy? I'm particually interested in OAuth providers multi tenancy support.

I have a SaaS application, and each tenant should have the ability to add their own OAuth or OID information from Google workspace and Microsoft 365 (my application currently supports those two, with plans to add Okta in the future).

If this is supported, can someone show me the documentation, I was not able to find this information.
A small sample project would be helpful if available.

thanks

**Comment by ashiva** [comment]
Same question here!
